pkg/nfd-worker/nfd-worker.go

```diff
@@ -239,6 +240,11 @@ func (w *nfdWorker) Run() error {
 	labelTrigger.Reset(w.config.Core.SleepInterval.Duration)
 	defer labelTrigger.Stop()
 
+	// Register to metrics server
+	go runMetricsServer(w.args.MetricsPort)
```

Comment:
Do we have input validation of MetricsPort somewhere? What if I specify `-metrics -1`. Should be disable with port < 1?